pkg/avx/collector.go

```diff
@@ -74,22 +75,21 @@ var descriptors = [numDescriptors]*prometheus.Desc{
 		AVXSwitchCountName,
 		"Number of task switches where AVX512 instructions were used in a particular cgroup.",
 		[]string{
-			"cgroup",
 			"cgroup_id",
 		}, nil,
 	),
 	allSwitchCountDesc: prometheus.NewDesc(
 		AllSwitchCountName,
 		"Total number of task switches in a particular cgroup.",
 		[]string{
-			"cgroup",
+			"cgroup_id",
```

Comment:
nitpick: isn't `cgroup_id` a bit misleading term ? IIRC, the original AVX code referred by cgroup ID to the uint64 handle/ID returned by unix.NameToHandleAt()/name_to_handle_at(2).

Shouldn't this be called `container_id` ? Or is this an established nomenclature among other existing upstream/external collectors ?

Reply:
I was thinking about `containerd_id` too so I'm ok with that.

Reply:
Ok. Since you later also call the regexp passed as an argument as containerIDRegexp, let's use `container_id` instead.